gallery/src/components/NxTabs/NxTabsTileHeaderExample.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel
+} from '@sonatype/react-shared-components';
+
+const NxTabsSimpleExample = () => {
+  const [activeTabId, setActiveTabId] = useState<number>(0);
+
+  function handleTabSelect(index: number) {
+    setActiveTabId(index);
+  }
+
+  return (
+    <div className="nx-tile">
+      <div className="nx-tile-header">
+        <div className="nx-tile-header__title"><h2 className="nx-h2">Tile Header</h2></div>
+      </div>
+      <div className="nx-tile-content">
+        <NxTabs activeTab={activeTabId} onTabSelect={handleTabSelect}>
+          <NxTabList aria-label="Simple Tabs">
+            <NxTab data-text="Tab">Tab</NxTab>
+            <NxTab data-text="Tab with longer name">Tab with longer name</NxTab>
+            <NxTab data-text="Another Tab 3">Another Tab 3</NxTab>
+            <NxTab data-text="Forth Tab">Forth Tab</NxTab>
+          </NxTabList>
+          <NxTabPanel>Tab 1</NxTabPanel>
+          <NxTabPanel>Tab 2</NxTabPanel>
+          <NxTabPanel>Tab 3</NxTabPanel>
+          <NxTabPanel>Tab 4</NxTabPanel>
+        </NxTabs>
+      </div>
```

Comment:
Should probably have a footer in at least one of the tile examples.

Reply:
> Should probably have a footer in at least one of the tile examples.

https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/cda6e6e96355d6fbfe70cdbc3722c0e6835f934a